src/material/chips/chip-list.ts

```diff
@@ -328,7 +328,7 @@ export class MatChipList extends _MatChipListMixinBase implements MatFormFieldCo
   @Output() readonly valueChange: EventEmitter<any> = new EventEmitter<any>();
 
   /** The chip components contained within this chip list. */
-  @ContentChildren(MatChip) chips: QueryList<MatChip>;
+  @ContentChildren(MatChip, {descendants: true}) chips: QueryList<MatChip>;
```

Comment:
Can you add a (non-JsDoc) comment here that explains why this has `{descendants: true}`?

Reply:
Done.